package.json

```diff
@@ -307,18 +307,19 @@
         "ts-node": "^7.0.1",
         "typescript": "^4.3.5",
         "vsce": "^1.87.0",
-        "vscode-azureextensiondev": "^0.10.0",
+        "vscode-azureextensiondev": "^0.10.4",
```

Comment:
Probably worth using the `@microsoft/vscode-azext-dev` package I just published.